packages/server/src/db/index.ts

```diff
@@ -1,12 +1,13 @@
+import { DataSource } from 'typeorm'
+import * as bcrypt from 'bcryptjs'
```

Comment:
Why use a JS implementation when node has a native one?

Reply:
oh node has native scrypt 🤔 
https://nodejs.org/api/crypto.html#crypto_crypto_scrypt_password_salt_keylen_options_callback

Reply:
oh my b. Is bcrypt that much better? Just gonna suck resources to do it in js

Reply:
no looks like scrypt is actually better. ill switch it over 👌